packages/three-vrm-springbone/src/VRMSpringBoneLoaderPlugin.ts

```diff
@@ -25,7 +25,7 @@ const POSSIBLE_SPEC_VERSIONS = new Set(['1.0', '1.0-beta']);
 /**
  * Possible spec versions of `VRMC_springBone_extended_collider` it recognizes.
  */
-const POSSIBLE_SPEC_VERSIONS_EXTENDED_COLLIDERS = new Set(['1.0-draft']);
+const POSSIBLE_SPEC_VERSIONS_EXTENDED_COLLIDERS = new Set(['1.0']);
```

Comment:
Since we don't expect to see much of `1.0-draft` models, we will drop support `1.0-draft` at this time.